tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>Alternative chart configuration using a "base" chart, that the points and fit charts inherit from.</p><p>(I think this is a bit easier to read and understand, but I don't feel very strongly about it.)</p><p> </p><p> </p><p><code>base = alt.Chart(ndvi_pdsi_df).encode(</code></p><p><code>    x=alt.X('PDSI:Q', scale=x_scale)</code></p><p><code>)</code></p><p> </p><p><code>points = base.mark_circle(size=60).encode(</code></p><p><code>    y=alt.Y('NDVI:Q', scale=y_scale),</code></p><p><code>    color=alt.Color('Year:O', scale=alt.Scale(scheme='magma')),</code></p><p><code>    tooltip=['Year', 'PDSI', 'NDVI'])</code></p><p> </p><p><code>fit = base.mark_line().encode(</code></p><p><code>    y=alt.Y('Fit:Q', scale=y_scale),</code></p><p><code>    color=alt.value('#808080'))</code></p><p> </p><p><code>(fit + points).properties(width=600, height=300).interactive()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Reply:
<p>This is great, I've reused the "base" configuration throughout. Done.</p>